pandas/tests/indexing/test_loc.py

```diff
@@ -2541,6 +2541,36 @@ def test_loc_with_period_index_indexer():
     tm.assert_frame_equal(df, df.loc[list(idx)])
 
 
+def test_loc_getitem_multiindex_tuple_level():
```

Comment:
does this cover the cases by @attack68 in the comment?

Reply:
only one of them i think